pkg/controller/replication_controller_test.go

```diff
@@ -183,10 +212,11 @@ func TestSyncReplicationControllerDeletes(t *testing.T) {
 	fakePodControl := FakePodControl{}
 
 	manager := NewReplicationManager(client)
+	close(manager.stopCh)
```

Comment:
To stop the reflector from updating the store, i mean it isn't 100% necessary for this test because I'm swapping the store out anyway, but it's necessary for any test that creates a replication controller with a fakeHandler and then does fakeHandler.Add() (eg TestWatchControllers), becuase the same fakehandler channel is being listened to by 2 things (the rc watcher and pod watcher, unless you explicitly give the rc watcher another channel, but doing this made it cleaner since I can also escape having to mock out all the list/ watch/ endpoints).